examples/basic-otlp-http/README.md

```diff
@@ -2,9 +2,11 @@
 * Run the application locally, to run as a docker container you have to change the relative paths from the `Cargo.toml`
 * The Collector then sends the data to the appropriate backend, in this case JAEGER
 
-This demo uses `docker-compose` and by default runs against the `otel/opentelemetry-collector-dev:latest` image.
+This demo uses `docker-compose` and by default runs against the `otel/opentelemetry-collector-dev:latest` image,
+and uses `http` as the transport.
 
 ```shell
+# in the `examples/basic-otlp/` directory:
```

Comment:
I think it maybe worth the copy the Dockerfile/docker-compose file/collector config file in this `examples/basic-otlp-http` too. 

The reasons its kind of intuitive to switch to another folder and then switch back to run the main application. Also docker related files doesn't really change overtime so it shouldn't be too hard to keep them in sync.

Reply:
good point! i pushed a new commit that does just this